app/src/main/java/org/p2p/wallet/home/events/EthereumTokensLoader.kt

```diff
@@ -39,22 +39,20 @@ class EthereumTokensLoader(
 
     private val state: MutableStateFlow<EthTokenLoadState> = MutableStateFlow(EthTokenLoadState.Idle)
 
+    // Caching last ethereum tokens, to prevent hiding tokens while refreshing
+    private val ethereumTokens = mutableListOf<Token.Eth>()
```

Comment:
I need to cache last eth tokens, to prevent hiding eth tokens, while sol tokens already loaded, but ethereum is not yet

Reply:
lastLoadedEthTokens